packages/components/src/components/popover/popover.jsx

```diff
@@ -194,7 +194,7 @@ const Popover = ({ ...props }) => {
                     <div
                         className={classNames('dc-popover', className)}
                         id={id}
-                        onMouseEnter={toggleOpen}
+                        onMouseOver={toggleOpen}
                         onMouseLeave={toggleClose}
```

Comment:
Can we use the `useHover` hook instead? It's used by `ThemedScrollbars` already and also uses `mouseOver` and `mouseOut`.

Reply:
I replaced this implementation with useHover hook